spec/models/shareholder_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Shareholder, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
maybe you can delete this file now (because there are no tests here anyway) and add it when you will have some tests